### DISCUSSION TITLE
Blocking Status button not working properly

### DISCUSSION THREAD

**Discussion by SLCCL** [0xERR0R/blocky#407] [Q&A] [answered]
Hello guys, it's me again.
Thanks for the advice and suggestion makes me can run "Blocky" on my raspberry pi. 
But there is still have a little problem that I've met (pic shows what happen when I hit the change status button). The status changing button not work for me.
Any suggestion would be appreciated. Thanks!

[https://i.postimg.cc/Dztjwcn0/011.png](url)

Here is the code of my docker-compose.yml

"me$ cat docker-compose.yml
version: "2.1"

services:
  blocky:
    image: spx01/blocky
    container_name: blocky
    restart: always
    ports:
      - "53:53/tcp"
      - "53:53/udp"
      - "4000:4000"
    volumes:
      - ./blocky/config.yml:/app/config.yml
  prometheus:
    image: prom/prometheus
    container_name: prometheus
    volumes:
      - ./prometheus/:/etc/prometheus/
      - prometheus_data:/prometheus
    command:
      - '--config.file=/etc/prometheus/prometheus.yml'
      - '--storage.tsdb.path=/prometheus'
      - '--web.console.libraries=/usr/share/prometheus/console_libraries'
      - '--web.console.templates=/usr/share/prometheus/consoles'
    ports:
      - 9090:9090
    restart: always
  grafana:
    image: grafana/grafana:8.3.3
    container_name: grafana
    depends_on:
      - prometheus
    ports:
      - 3000:3000
    volumes:
      - grafana_data:/var/lib/grafana
      - ./grafana/provisioning/:/etc/grafana/provisioning/
    environment:
      - GF_PANELS_DISABLE_SANITIZE_HTML=true
      - GF_INSTALL_PLUGINS=grafana-piechart-panel
    restart: always

volumes:
    prometheus_data: {}
    grafana_data: {}
"

**Comment by 0xERR0R** [comment, marked as answer]
if you import the grafana dashboard, you must set blocky's REST url. This should be the ip of your raspberry and port 4000. Something like: http://raspberry_ip:4000
Can you check it please?

**Reply by SLCCL** [reply]
You're right! Thanks a lot.
I found there is "Http://localhost:4000" on the dashboard's value.
It works properly after change to "http://raspberry_ip:4000".

Now it's time to move on next task - Query Log
